cirq-google/cirq_google/optimizers/optimize_for_xmon.py

```diff
@@ -21,17 +21,19 @@
     import cirq_google
 
 
+@cirq._compat.deprecated_parameter(
+    deadline='v0.15',
+    fix=cirq.circuits.circuit._DEVICE_DEP_MESSAGE,
+    parameter_desc='new_device',
+    match=lambda args, kwargs: 'new_device' in kwargs,
+)
 def optimized_for_xmon(
```

Comment:
Plan to deprecate the device function signature from this optimizer because it is not a device aware optimizer and all the device was used for was attaching it to the returned circuit.